library/aes.c

```diff
@@ -510,6 +510,8 @@ static void aes_gen_tables( void )
 
 void mbedtls_aes_init( mbedtls_aes_context *ctx )
 {
+    MBEDTLS_AES_VALIDATE( ctx != NULL );
```

Comment:
Please note that there are actually a lot of other functions that are also missing parameter checks.

Reply:
As above. This was written as a PoC, and not needs to be expanded.